docs/source/en/using-diffusers/loading.mdx

```diff
@@ -178,105 +198,177 @@ stable_diffusion_img2img = StableDiffusionImg2ImgPipeline(**components)
 
 Note how the above code snippet makes use of [`DiffusionPipeline.components`].
 
+### Loading variants
+
+Diffusion Pipeline checkpoints can offer variants of the "main" diffusion pipeline checkpoint.
+Such checkpoint variants are usually variations of the checkpoint that have advantages for specific use-cases and that are so similar to the "main" checkpoint that they **should not** be put in a new checkpoint. A variation of a checkpoint has to have **exactly** the same checkpoint layout including all weights having the same tensor shapes.
+
+#### Let's first talk about what checkpoint variants are **not**.
+
+Checkpoint variants do **not** include different serialization format (such as [safetensors](https://huggingface.co/docs/diffusers/main/en/using-diffusers/using_safetensors)) as different serialization formats are 
+identical to the "main" serialization format, just loaded with different code.
+
+Also variants do not correspond to different checkpoint layouts, *e.g.* [stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5) is not a variant of [stable-diffusion-2-0](https://huggingface.co/stabilityai/stable-diffusion-2) since the checkpoint layout is different (Stable Diffusion 1-5 uses a different `CLIPTextModel` compared to Stable Diffusion 2.0).
+
+Pipeline checkpoints that are identical in checkpoint layout, but have been trained on different datasets, trained with vastly diffenent training setups and thus correspond to different official releases (such as [Stable Diffusion v1-4](https://huggingface.co/CompVis/stable-diffusion-v1-4) and [Stable Diffusion v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5)) should probably also not be stored as variations of each other, but rather as individual model repositories.
+
+#### So what are checkpoint variants then? 
+
+Checkpoint variants usually consists of the checkpoint stored in "*low-precision, low-storage*" dtype so that less bandwith is required to download them or of *non-exponential-averaged* weights that shall be used when continuing fine-tuning the checkpoint.
```

Comment:
```suggestion
Checkpoint variants usually consist of the checkpoint stored in "*low-precision, low-storage*" dtype so that less bandwith is required to download them, or of *non-exponential-averaged* weights that shall be used when continuing fine-tuning from the checkpoint.
```